utils/preDeployContracts.js

```diff
@@ -0,0 +1,53 @@
+const { deployContracts } = require('./deployContracts');
+
+const fs = require('fs-extra');
+
+module.exports = {
+  preDeployContracts: function(buildContractsPath, appPreBuiltContractArtifactsPath) {
+    fs.emptyDirSync(buildContractsPath, '');
+    process.env.TARGET_NETWORK = 'ropsten';
+    console.log('deploying to ropsten');
+    deployContracts()
+      .then(() => {
+        process.env.TARGET_NETWORK = 'kovan';
+        console.log('deploying to kovan');
+        return deployContracts();
+      })
+      .then(() => {
+        process.env.TARGET_NETWORK = 'rinkeby';
+        console.log('deploying to rinkeby');
+        return deployContracts();
+      })
+      .then(() => {
```

Comment:
Might be cleaner to write this utility using `async` + `await` to avoid `.then` nesting.

Reply:
Agreed, but I'll leave this for a future PR

Reply:
In fact, I addressed this in https://github.com/magmo/devtools/pull/22/commits/8ddb102f335e230f9fd6e78f055c5c929ef2b9f3 . There's still one `.then` but I think it is an improvement overall.